x/docs/md/track/TRY.md

```diff
@@ -15,8 +15,15 @@ exercism configure --key=YOUR_EXERCISM_KEY
 ```
 
 When you've written a solution, submit it to the site.
-You'll have to configure the command-line client with your exercism API key before you can submit.
+The solution is a source file, in wich you have written your code
+to pass the test suite of a given problem, so the command to submit it will look like this:
 
 ```plain
-exercism submit PATH_TO_FILE
+exercism submit PATH_TO_YOUR_SOURCE_FILE
 ```
+
+If you have splitted your solution in several files or have used additional configuration files, submit them too.
```

Comment:
I think we can just use `split` here instead of ` have splitted`. Also change `in` to `into`